wandb/apis/public.py

```diff
@@ -4235,9 +4235,30 @@ def id(self):
     def file_count(self):
         return self._attrs["fileCount"]
 
+    @property
+    def source_version(self):
+        """
+        Returns:
+            (str) The artifact's version index under its parent artifact collection. This will return
+            a string with the format "v{number}".
+        """
+        return f"v{self._sequence_version_index}"
+
     @property
     def version(self):
-        return "v%d" % self._version_index
+        """
+        Returns:
+            (str): The artifact's version index under the given artifact collection. This will return
+            a string with the format "v{number}".
+        """
+        for a in self._attrs["aliases"]:
+            if a[
+                "artifactCollectionName"
+            ] == self._artifact_collection_name and util.alias_is_version_index(
+                a["alias"]
```

Comment:
same for these keys

Reply:
Yes, `artifactCollectionName` is always guaranteed to exist.